### DISCUSSION TITLE
Welcome to mikan_flutter Discussions!

### DISCUSSION THREAD

**Discussion by iota9star** [iota9star/mikan_flutter#17] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LiYunDong-bilibili** [comment]
笔记本使用桌面端软件 图片模糊

**Reply by iota9star** [reply]
暂时可能不会解决，引擎问题。如果引擎长时间未解决我可能才会尝试去解决
